sagemaker-python-sdk/tensorflow_script_mode_pipe_mode/tensorflow_script_mode_pipe_mode.ipynb

```diff
@@ -140,7 +140,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "After we've created the SageMaker Python SDK TensorFlow object, we can call fit to launch TensorFlow training:"
+    "After we've created the SageMaker Python SDK TensorFlow object, we can call ``fit`` to launch TensorFlow training:"
```

Comment:
I'd be consistent between here and above for if you're including parentheses when writing about functions